### PR TITLE
DOCAPI-6181: RU translation for Parquet docs.

### PR DESCRIPTION
**Pull request by BayoNet**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Category (leave one):
- Documentation for #4448

### CODE REVIEW

File: docs/en/interfaces/formats.md [hunk 1]
```diff
@@ -758,8 +758,7 @@ You can select data from a ClickHouse table and save them into some file in the
 clickhouse-client --query="SELECT * FROM {some_table} FORMAT Parquet" > {some_file.pq}
 ```
 
-Also look at the `HDFS` and `URL` storage engines to process data from the remote servers.
-
+To exchange data with the Hadoop ecosystem, you can use `HDFS` and `URL` table engines.
```
Comment (blinkov) [inline docs/en/interfaces/formats.md]: * URL engine has nothing to do with Hadoop, so this sounds weird.
* "ecosystem" is too wide term, `HDFS` table engine and function work specifically with Hadoop and it's filesystem (which is called HDFS)
* HDFS+Parquet needs an example.
* "available to any project in the Hadoop ecosystem" above sounds weird, it's just one of a widespread formats in that ecosystem

**Review by blinkov — CHANGES_REQUESTED** [review]

File: docs/en/interfaces/formats.md (hunk 1 above)
Comment (BayoNet) [reply docs/en/interfaces/formats.md]: - URL engine allows to retrieve data from Hadoop through its HTTP interface?
- I'll do this example, when describing HDFS table engine.

File: docs/en/interfaces/formats.md [hunk 2]
```diff
@@ -758,8 +758,7 @@ You can select data from a ClickHouse table and save them into some file in the
 clickhouse-client --query="SELECT * FROM {some_table} FORMAT Parquet" > {some_file.pq}
 ```
 
-Also look at the `HDFS` and `URL` storage engines to process data from the remote servers.
-
+To exchange data with the Hadoop, you can use `HDFS` and `URL` table engines.
```
Comment (blinkov) [inline docs/en/interfaces/formats.md]: links?

File: docs/en/interfaces/formats.md (hunk 1 above)
Comment (blinkov) [reply docs/en/interfaces/formats.md]: > URL engine allows to retrieve data from Hadoop through its HTTP interface?

Afaik it's not a common usage pattern for HDFS, available only via some extension. URL is more applicable to S3, but dedicated support for it is also coming soon in #5596

File: docs/en/interfaces/formats.md [hunk 3]
```diff
@@ -758,7 +758,7 @@ You can select data from a ClickHouse table and save them into some file in the
 clickhouse-client --query="SELECT * FROM {some_table} FORMAT Parquet" > {some_file.pq}
 ```
 
-To exchange data with the Hadoop, you can use `HDFS` and `URL` table engines.
+To exchange data with the Hadoop, you can use `HDFS` table engines.
```
Comment (blinkov) [inline docs/en/interfaces/formats.md]: table engine--s--

**Pull request merged**